### ISSUE TITLE
Slider is not operable when using VoiceOver on iOS

### ISSUE THREAD

**Issue by mfairchild365** [angular/components#15037]
#### What is the expected behavior?
It must be possible to adjust the value of the slider component while using VoiceOver on iOS.

#### What is the current behavior?
It is not possible to adjust the value of the slider component while using VoiceOver on iOS. 

#### What are the steps to reproduce?
1. Using VoiceOver for IOS (12.1.3) + Safari
2. Open the [slider example](https://material.angular.io/components/slider/examples)
3. Navigate to one slider the example
   - In VoiceOver for iOS you can swipe right until you reach the slider
4. Observe that Voice Over announces "0%, adjustable. Swipe up or down with one finger to adjust the value"
5. Swipe up or down to adjust the value
6. Observe that the value was not changed

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
- Angular Material Example 7.2.2 located at https://material.angular.io/components/
- VoiceOver for IOS (12.1.3) + Safari (12.1.3)

#### Is there anything else we should know?
To my knowedge, it is not currently possible for JS to listen to the events fired by VoiceOver to adjust the value. This also likely applies to other touch-based screen readers, including Talkback for android. There is a [relevent discussion about this in the aria-practices repo](https://github.com/w3c/aria-practices/issues/8). This issue will likely be addressed by the future AOM standard, which creates several [new inputEvent types](https://github.com/WICG/aom/blob/gh-pages/explainer.md#new-inputevent-types).

Until then, it might be possible to use an [html `range` input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range) in most situations.

**Comment by cyrilledjdj** [comment]
Is anyone still interested for a solution to this issue?

**Comment by jelbourn** [comment]
Status: we're working on a new version of the slider based on MDC Web that should fix this issue by using a native `<input type="range">` in the template

**Comment by wagnermaciel** [comment]
@jelbourn is it possible to test this in our unit or e2e tests?

**Comment by jelbourn** [comment]
@wagnermaciel no, automated tests for screen-readers aren't really a thing yet.

**Comment by amysorto** [comment]
The MDC based slider is now available (starting in v15), although this is still an issue when I checked.

**Comment by wagnermaciel** [comment]
Does this issue happen on a native html slider as well? Because if so, this can be closed as it should be considered a bug with VoiceOver

**Comment by amysorto** [comment]
Nope, inputs by default have the [step value set to 0 by default](https://github.com/angular/components/blob/main/src/material/slider/slider.ts#L287). In [the basic slider example](https://material.angular.io/components/slider/examples#slider-overview), the step value is 0 which is why it wasn't working for me. While that works on web with keyboard navigation, it disables VoiceOver from being able to move the slider up and down. It does work correctly when step is 1 or higher. Maybe the default step value should be 1? @wagnermaciel
